requirements/base.in

```diff
@@ -7,9 +7,10 @@ ipython==8.9.0
 # The following was generated by 'tox -e deps', DO NOT EDIT MANUALLY!
 dask
 graphviz
+essreduce
 plopp
 pythreejs
-sciline>=24.2.0
+sciline>=23.9.1
```

Comment:
I don't know, @jl-wynen ?

Reply:
It happened in this commit: https://github.com/scipp/esssans/pull/114/commits/e7575250c6ecdd3b5ce870519ba8879f5994311b because I updated dependencies. This replaced the bound because it was set in https://github.com/scipp/esssans/commit/825f4adb05d6c9fe773a9f15b877ba02e2411c7f in `base.in` instead of `pyproject.toml`.

Reply:
ok I fixed it in the `pyproject.toml`.